quick-start-with-tidb.md

```diff
@@ -6,25 +6,125 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
+    source .bash_profile
+    ```
+
+3. Start the cluster in the current session:
+
+    - If you want to start a TiDB cluster of the latest version with 1 TiDB instance, 1 TiKV instance, 1 PD instance, and 1 TiFlash instance, run the following command:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup playground
+        ```
+
+    - If you want to specify the TiDB version and the number of the instances of each component, run a command like this:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup playground v5.0.1 --db 2 --pd 3 --kv 3 --monitor
+        ```
+
+        The command downloads a version cluster to the local machine and starts it, such as v5.0.1. `--monitor` means that the monitoring component is also deployed.
+
+        To view the latest version, run `tiup list tidb`.
+
+        This command returns the access methods of the cluster:
+
+        ```log
+        CLUSTER START SUCCESSFULLY, Enjoy it ^-^
+        To connect TiDB: mysql --host 127.0.0.1 --port 4000 -u root
+        To connect TiDB: mysql --host 127.0.0.1 --port 4001 -u root
+        To view the dashboard: http://127.0.0.1:2379/dashboard
+        To view the monitor: http://127.0.0.1:9090
+        ```
+
+        > **Note:**
+        >
+        > For the playground operated in this way, after the test is finished, TiUP will clean up the original cluster data. You will get a new cluster after re-running the command.
+        > If you want the data to be persisted on storage，run `tiup --tag <your-tag> playground ...`. For details, refer to [TiUP Reference Guide](/tiup/tiup-reference.md#-t---tag).
+
+4. Start a new session to access TiDB:
 
-This document describes how to try out TiDB, a distributed SQL database, in a few minutes. You can choose any of the following two methods to get started with the TiDB database platform:
+    + Use the TiUP client to connect to TiDBs.
 
-- [Deploy a local test environment using TiUP Playground](#deploy-a-local-test-environment-using-tiup-playground)
-- [Set up a test environment on a single machine using TiUP Cluster](#set-up-a-test-environment-on-a-single-machine-using-tiup-cluster)
+        {{< copyable "shell-regular" >}}
 
-## Deploy a local test environment using TiUP Playground
+        ```shell
+        tiup client
+        ```
 
-- Scenario: Quickly deploy a TiDB cluster on a local Mac or Linux OS machine. You can see the basic architecture of TiDB and how TiDB, TiKV, PD, monitoring, and other components run in a TiDB cluster.
-- Time required: 1 minute
+    + You can also use the MySQL client to connect to TiDB.
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        mysql --host 127.0.0.1 --port 4000 -u root
+        ```
+
+5. Access the Prometheus dashboard of TiDB at <http://127.0.0.1:9090>.
+
+6. Access the [TiDB Dashboard](/dashboard/dashboard-intro.md) at <http://127.0.0.1:2379/dashboard>. The default username is `root`, with an empty password.
```

Comment:
Maybe we should add Grafana. I think that uses `admin/admin` as default credentials, not sure if that's included in this doc or the `tiup playground` output.

Reply:
This PR aims to solve some specific issues and is large enough for reviewers. To merge this PR quickly, I suggest we keep each PR relatively small. 

For new ideas for improving this page, may I suggest that we discuss and apply them in another new PR? I added a "requires-followup" label to follow up on your suggestions after we merge this PR.

Reply:
Lets add it later. It doesn't make sense to have the prometheus dashboard and not grafana.